test/solidity-ibc/IntegrationTest.t.sol

```diff
@@ -90,14 +102,14 @@ contract IntegrationTest is Test {
 
         assertEq(address(ics20Transfer), address(ics26Router.getIBCApp(ICS20Lib.DEFAULT_PORT_ID)));
 
-        defaultSender = makeAddr("sender");
+        (defaultSender, defaultSenderKey) = makeAddrAndKey("sender");
         defaultSenderStr = Strings.toHexString(defaultSender);
 
         defaultReceiver = makeAddr("receiver");
         defaultReceiverStr = Strings.toHexString(defaultReceiver);
     }
 
-    function test_success_sendICS20PacketDirectlyFromRouter() public {
```

Comment:
The diff looks a bit weird, but I essentially deleted this test because it was doing the same thing as the next one. And then I added a new test.